spyder/workers/updates.py

```diff
@@ -27,16 +27,21 @@
 # Logger setup
 logger = logging.getLogger(__name__)
 
+CONNECT_ERROR_MSG = _(
+    'Unable to connect to the Spyder update service.'
+    '<br><br>Make sure your connection is working properly.'
+)
+
+HTTP_ERROR_MSG = _(
+    'HTTP error {status_code} when checking for updates.'
+    '<br><br>Make sure your connection is working properly.'
```

Comment:
```suggestion
    '<br><br>Make sure your connection is working properly,'
    'and try again later.'
```

This could also just be a transient issue on the server end, so would be good to mention something like this—wanted to include it before, but wasn't sure what to say.